plasmapy/formulary/parameters.py

```diff
@@ -77,6 +77,7 @@
     ("frequencies", "wlh_"),
     ("frequencies", "upper_hybrid_frequency"),
     ("frequencies", "wuh_"),
+    ("frequencies", "plasma_critical_density"),
```

Comment:
Is this necessary?

Reply:
No I don't think so - this is just an artifact of the way we use to have this organized: all these functions were in `parameters.py`. This code is just for backward compatibility for a few versions. Since your function is new, it doesn't need to go here.